h/services/search_index.py

```diff
@@ -48,7 +48,7 @@ def add_annotation_by_id(self, annotation_id):
         if annotation.is_reply:
             self.add_annotation_by_id(annotation.thread_root_id)
 
-    def add_annotation(self, annotation):
+    def add_annotation(self, annotation, refresh=False):  # noqa: FBT002
```

Comment:
Include this here to use True on the new endpoint

https://elasticsearch-py.readthedocs.io/en/v8.18.1/api/elasticsearch.html#elasticsearch.Elasticsearch.index


``` If true, Elasticsearch refreshes the affected shards to make this operation visible to search.```

Reply:
Yes, good point.